mypyc/genops.py

```diff
@@ -2075,14 +2075,58 @@ def translate_call(self, expr: CallExpr, callee: Expression) -> Value:
 
     def translate_refexpr_call(self, expr: CallExpr, callee: RefExpr) -> Value:
         """Translate a non-method call."""
-        # Gen the argument values
-        arg_values = [self.accept(arg) for arg in expr.args]
 
         # TODO: Allow special cases to have default args or named args. Currently they don't since
         # they check that everything in arg_kinds is ARG_POS.
 
         # TODO: Generalize special cases
 
+        def gen_any_all_inner_stmts(comparison: Value, op: OpDescription) -> None:
+            true_block, false_block = BasicBlock(), BasicBlock()
+            self.add_bool_branch(comparison, true_block, false_block)
+            self.activate_block(true_block)
+            self.assign(retval, self.primitive_op(op, [], -1), -1)
+            self.nonlocal_control[-1].gen_break(self)
```

Comment:
Unfortunately, using break won't work for nested generator comprehensions.

Also please pull all this logic out into another function.